contents/blog/stop-treating-engineers-differently.md

```diff
@@ -0,0 +1,68 @@
+
+## Engineers are motivated by different goals 
+
+Having conducted about ~800 job interviews, it turns out people by and large tend to say the same 3 things:
+
+- _Autonomy_ – freedom to make decisions and not be micromanaged
+- _Purpose_ – working for a company or team I care about
+- _Growth_ – feeling like I’m learning and developing
+
+This whole simplistic ‘commercial people only want money and titles, engineers are happy building delightful products in a dungeon’ doesn’t stack up. 
+
+**Why this matters:** It causes startups to underinvest in their engineers’ development (this is more prevalent in those that see engineering as a [cost vs. profit center](https://newsletter.pragmaticengineer.com/p/profit-centers-cost-centers)). There’s a reason even ~~FAANG~~ MAMAA companies struggle with engineer retention, and it’s not just about $$$. 
+
+## Engineers hate meetings
+
+This should be reframed as ‘everyone thinks they hate meetings because 90% of the ones they attend are a waste of time’. Do you really think the junior sales rep sitting on a Zoom call with 30 other people giving status updates is excited to be there?
+
+Meetings are a really important tool, [especially for managers](https://www.goodreads.com/book/show/324750.High_Output_Management) (which is why they love them), but just like any tool they can be ineffectively used. 
+
+**Why this matters:** It lets startups get away with packing their weeks with ineffective meetings for non-engineers. At PostHog at least, being mildly zealous about meeting-free Tuesdays and Thursdays means a) minimal low value meetings, and b) better communication because you’re forced to write stuff down properly. 
+
+## Engineers are allergic to process
+
+Ah yes, non-engineers love writing policies and documentation, while engineers just want to YOLO merge in peace. Please. 
+
+And no, calling your corporate policies ‘runbooks’ doesn’t make them any more palatable. 
+
+The reality is, _every_ role has people who are process-driven and those who aren’t (wow, we’re really laboring the same point here, aren’t we…).
+
+**Why this matters:** It leads to lazy process-making overall at a company. Instead of creating thoughtful processes that improve the lives of your team, you end up copying and pasting something you found online, announcing it at all hands, and then accepting that engineers won’t follow it because ‘they’re engineers’. 
+
+## Enough ranting... what should you do about it?
+
+As a non-engineer, my one simple trick™ is to treat engineers as cultural canaries in the coal mine. 
+
+Every time you hear someone saying ‘engineers don’t like X’ or ‘the engineering team have complained about Y’, **use that as an opportunity to think about how this applies to your whole organization**. Nine times out of 10, I’ll bet you find that you’re looking at a broader team culture issue, not an engineering one. 
```

Comment:
I know this is more of a rant than a guide, but if possible I'd like to expand on this conclusion a little more.

Reply:
I added a bit more, what do you think?

Reply:
Yeah, this is perfect.